depthcharge/data/parsers.py

```diff
@@ -479,7 +482,10 @@ def _parse_scan_id(scan_str: str | int) -> int:
         try:
             return int(scan_str[scan_str.find("scan=") + len("scan=") :])
         except ValueError:
-            pass
+            try:
```

Comment:
codecov has a point ... this should be tested ... This has hurt me before in scans that have both "scan=" and "index=" and they are off by one ...
make sure your behavior is consistent.

Reply:
Good point!